libs/server/shared/src/services/market-data.service.ts

```diff
@@ -39,7 +39,7 @@ export interface IMarketDataService {
     /**
      * fetches pricing info for inclusive date range
      */
-    getDailyPricing<TSecurity extends Pick<Security, 'symbol' | 'plaidType' | 'currencyCode'>>(
```

Comment:
does currencyCode need to go? looks like it might be required for the polygon api calls, but could be mistaken